src/main/java/com/github/vkorobkov/jfixturescmd/CmdParser.java

```diff
@@ -0,0 +1,43 @@
+package com.github.vkorobkov.jfixturescmd;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.ParameterException;
+import com.github.vkorobkov.jfixtures.JFixtures;
+import com.github.vkorobkov.jfixtures.fluent.JFixturesResultImpl;
+import com.github.vkorobkov.jfixturescmd.utils.PropertiesReader;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+
+import java.lang.reflect.Method;
+
+@Slf4j
+final class CmdParser {
+    private final CmdArgs cmdArgs = new CmdArgs();
```

Comment:
Can we have `CmdArgs` as a singleton so all the input parameters will be globally available?